spec/javascript/pages/browse/ContactIcons.spec.js

```diff
@@ -1,11 +1,14 @@
-import Vue from 'our_vue'
-import {shallowMount} from '@vue/test-utils'
+import {configure} from 'vue_config'
+import {createLocalVue, shallowMount} from '@vue/test-utils'
 import ContactIcons from 'pages/browse/ContactIcons'
 
-it('generally works', function() {
-  const wrapper = shallowMount(ContactIcons, {
-    propsData: { contactTypes: [{id: 1, name: "email"}, {id: 2, name: "text"},{id: 3, name: "call"}]}
+describe('ContactIcons', () => {
+  it('generally works', function() {
+    const wrapper = shallowMount(ContactIcons, {
+      localVue: configure(createLocalVue()),
```

Comment:
`vue-test-utils` pattern for ensuring we don't pollute a shared global Vue instance between specs.